README.md

```diff
@@ -1,15 +1,111 @@
 # LocalNetAppChat
 
-Next tasks could be: 
+LocalNetAppChat (LNAC) is a server/client command line interface that gives your apps a way to communicate with each other localy.
 
-- [ ] add documentation
+<!-- TOC -->
+
+- [Features](#features)
+- [Installation](#installation)
+  - [Server](#server)
+  - [Client](#client)
+- [Usage](#usage)
+  - [Server CLI](#server-cli)
+  - [Client CLI](#client-cli)
+- [ToDo](#todo)
+- [Contributions](#contributions)
+- [Questions?](#questions?)
+
+<!-- /TOC -->
+
+## Features
+
+- Available for Windows, Linux and macOS
+- Easy to use
+- Direct Communication between apps
```

Comment:
Adds a few simple entry points / command line tools that enable network communication between a group of hosts and sub applications.